exchange/exchange.go

```diff
@@ -553,8 +555,13 @@ func (e *exchange) getAllBids(
 		brw := <-chBids
 
 		//if bidder returned no bids back - remove bidder from further processing
-		if brw.adapterBids != nil && len(brw.adapterBids.bids) != 0 {
-			adapterBids[brw.bidder] = brw.adapterBids
+		for _, seatBid := range brw.adapterSeatBids {
+			if seatBid != nil && len(seatBid.bids) != 0 {
+				// Do we need to address duplicate names here?. What if different BidderNames have exta bids with same seat value.
+				// Ex. 'pubmatic' adapter has extra bids under new 'groupm' seat and 'appnexus' also comes with additional bids under seat 'groupm'
+				// Above example might be more relatable with 'allowUnknownBidderCodes'
+				adapterBids[openrtb_ext.BidderName(seatBid.seat)] = seatBid
```

Comment:
Do we need to address duplicate names here?

Reply:
Do we clubs the bids in case multiple bidders returns the same alternate bidder code?. Ex. `appnexus` and `Pubmatic` returning `groupm` as a new bidder

@bretg thoughts?

Reply:
updated code to append the bids